src/assets/sitemapper.js

```diff
@@ -203,49 +223,99 @@ export default class Sitemapper {
    * @private
    * @recursive
    * @param {string} url - the Sitemaps url (e.g https://wp.seantburke.com/sitemap.xml)
+   * @param {integer} retryIndex - Number of retry attempts fro this URL (e.g. 0 for 1st attempt, 1 for second attempty etc.)
    * @returns {Promise<SitesArray> | Promise<ParseData>}
```

Comment:
@bsq-panagiotis The return signature is actually changing here, so might need to make this a major version bump to 4.0.0 as it will break current integrations expecting an array.

Reply:
Resolved [here](https://github.com/seantomburke/sitemapper/pull/78#discussion_r588693565)